drivers/memc/memc_mcux_flexspi.c

```diff
@@ -140,6 +140,8 @@ static int memc_flexspi_init(const struct device *dev)
 #define MEMC_FLEXSPI_CFG_XIP(node_id) DT_SAME_NODE(node_id, DT_NODELABEL(flexspi))
 #elif defined(CONFIG_XIP) && defined(CONFIG_CODE_FLEXSPI2)
 #define MEMC_FLEXSPI_CFG_XIP(node_id) DT_SAME_NODE(node_id, DT_NODELABEL(flexspi2))
+#elif defined(CONFIG_SOC_SERIES_IMX_RT6XX)
+#define MEMC_FLEXSPI_CFG_XIP(node_id) true
```

Comment:
It solely depends on the soc type here, but XIP is enabled in the board defconfig.

```suggestion
#define MEMC_FLEXSPI_CFG_XIP(node_id) IS_ENABLED(CONFIG_FLASH_MCUX_FLEXSPI_XIP)
```
Or
```suggestion
#define MEMC_FLEXSPI_CFG_XIP(node_id) IS_ENABLED(CONFIG_XIP)
```

Reply:
Done. Updated to `#define MEMC_FLEXSPI_CFG_XIP(node_id) IS_ENABLED(CONFIG_XIP)`